cadence/config/cadence.env

```diff
@@ -0,0 +1,16 @@
+CSERVER_MUSIC_DIR=/music/
+CSERVER_REQRATELIMIT=180
+
+# ####################################################
+# If you are running Cadence through Docker simply as a user, 
+# you are unlikely to ever need to change anything below.
+
+CSERVER_VERSION=5.0.0
```

Comment:
You're gonna hate me...

Couldn't this be replaced using git as a target in the makefile? Either the current tag or the short hash of the HEAD commit when you run the build?

I get the feeling that updating this is going to be mildly more annoying than having it done automagically.

... Plus we're engineers, and automating things is cool.

Reply:
Maybe it could. It does sound cleaner that way. I don't disagree that this current way could cause some irritation.